jcheck/src/main/java/org/openjdk/skara/jcheck/ReviewersConfiguration.java

```diff
@@ -130,31 +130,40 @@ static ReviewersConfiguration parse(Section s) {
         var contributors = s.get("contributors", 0);
 
         if (s.contains("minimum")) {
-            // Reset defaults to 0
-            lead = 0;
-            reviewers = 0;
-            committers = 0;
-            authors = 0;
-            contributors = 0;
-
-            var minimum = s.get("minimum").asInt();
-            if (s.contains("role")) {
-                var role = s.get("role").asString();
-                if (role.equals("lead")) {
-                    lead = minimum;
-                } else if (role.equals("reviewer")) {
-                    reviewers = minimum;
-                } else if (role.equals("committer")) {
-                    committers = minimum;
-                } else if (role.equals("author")) {
-                    authors = minimum;
-                } else if (role.equals("contributor")) {
-                    contributors = minimum;
+            var isMinimumDisabled = s.get("minimum").asString().trim().toLowerCase().equals("disable");
+            if (!isMinimumDisabled) {
+                for (var role : List.of("lead", "reviewers", "committers", "authors", "contributors")) {
+                    if (s.contains(role)) {
+                        throw new IllegalStateException("Cannot combine 'minimum' with '" + role + "'");
```

Comment:
Is there any risk that existing .jcheck/conf files will become invalid because of this new failure mode?

Reply:
Yes, but I have never seen a `.jcheck/conf` file with both `minimum` and e.g. `reviewers` set. I looked through all github.com/openjdk repositories and there are only three non-archived repos that has `minimum` set:
- [jmc](https://github.com/openjdk/jmc/blob/master/.jcheck/conf#L49)
- [jmc-graphics](https://github.com/openjdk/jmc-graphics/blob/master/.jcheck/conf#L48)
- [jfx11u](https://github.com/openjdk/jfx11u/blob/master/.jcheck/conf#L43)

None of the above has conflicting keys set, so with this patch we will be able to handle them all. So I think we are good here :+1: